.github/workflows/_lints.yml

```diff
@@ -0,0 +1,83 @@
+name: Reusable workflow for lints
+
+on:
+  workflow_call:
+    inputs:
+      target:
+        required: true
+        type: string
+
+jobs:
+  clippy:
+    name: Clippy (MSRV)
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ${{ inputs.target }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install build dependencies
+        run: sudo apt install libudev-dev
+      - name: Run Clippy
+        uses: actions-rs/clippy-check@v1
```

Comment:
If `actions-rs` is used anyway than maybe it is OK to use their `toolchain` action too.

My motivation to not use it in #82 was that this actions are not maintained and use some deprecated GH features (such as Node 12), but if at least one of them has to be used (`clippy-check`) then there is no good readon to omit the other ones.

Reply:
I kept using this one action because I had not yet found a viable alternative, but I just did: https://github.com/auguwu/clippy-action